### ISSUE TITLE
Removing unused headers and functions

### ISSUE THREAD

**Issue by ghenry** [GrumpyOldTroll/libmcrx#19]
Some of these includes can be deleted, but I'm presuming the unused functions are for users of the library?

![Screenshot from 2022-02-03 12-20-19](https://user-images.githubusercontent.com/159608/152341853-4bba580e-b764-4795-a63c-ac728844c1dd.png)

We could clear these warnings by using them in a test cases if possible.

**Comment by GrumpyOldTroll** [comment]
Yes, test cases that use these is I think the right action for those functions.

Was there a file for the unused includes?  Pretty sure those are needed in some cases, but I believe it if it's saying they're present in some places they're not needed.
